.github/workflows/fe2-android.yml

```diff
@@ -0,0 +1,38 @@
+name: PoP CI fe2-android
```

Comment:
this will need updating (at the same time as the other workflows) after the merge